build/AzurePipelinesTemplates/WindowsAppSDK-SetupBuildEnvironment-Steps.yml

```diff
@@ -147,6 +147,14 @@ steps:
     arguments: -Path $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-AppNotifications.xml -Channel $(channel) -Language C++ -Namespace Microsoft.Windows.AppNotifications -Output $(Build.SourcesDirectory)\dev\common\TerminalVelocityFeatures-AppNotifications.h
     workingDirectory: '$(Build.SourcesDirectory)'
 
+- task: powershell@2
+  displayName: 'Create CallingPreviewSupport TerminalVelocity features'
```

Comment:
What is `CallingPreviewSupport`? Shouldn't this be `AppNotifications`

Reply:
`CallingPreviewSupport` was created as a separate feature, which may make sense, but the way this was defined was a little weird, as discussed in your other comment.